### ISSUE TITLE
Error Reading WKT

### ISSUE THREAD

**Issue by jhudsonWA** [sibartlett/Geo#22]
Trying to read a WKT that is in a SRID (2927) and it keeps telling me:

Specified argument was out of the range of valid values.

Does this reader only work with regular lat/longs?

WKT: 
POINT (1231668.2539279764 968951.39716555178)

Code:

 var reader = new WktReader();
 var geometry = reader.Read(roadsideFeature.FeatureGeometryString); // reading a string

**Comment by sibartlett** [comment]
Yeah, this library only works with regular lat/longs; SRID 4326 / WGS84.

**Comment by jhudsonWA** [comment]
Ok, thanks anyway!
